src/coreComponents/common/format/table/TableData.cpp

```diff
@@ -18,21 +18,36 @@
  */
 
 #include "TableData.hpp"
+#include "common/logger/Logger.hpp"
 
 namespace geos
 {
 
-void TableData::addRow( std::vector< string > const & row )
+void TableData::addRow( std::vector< TableData::CellData > const & rows )
 {
-  if( m_rows.size() != 0 && row.size() != m_rows[m_rows.size() - 1].size() )
+  if( m_rows.size() != 0 && rows.size() != m_rows[m_rows.size() - 1].size() )
   {
     string msg = "Remarks : some cells may be missing";
+    std::cout << '\n';
```

Comment:
?

Reply:
I will remove this part, errors in table, like mismatching nb data in ro /header will be handle in future pr I think